internal/functions/deploy/deploy.go

```diff
@@ -34,6 +34,17 @@ func Run(ctx context.Context, slug string, projectRefArg string, verifyJWT bool,
 		} else if !utils.ProjectRefPattern.MatchString(projectRef) {
 			return errors.New("Invalid project ref format. Must be like `abcdefghijklmnopqrst`.")
 		}
+		// Ensure noVerifyJWT is not nil.
+		if noVerifyJWT == nil {
+			x := false
+			// Load function config if any, but continue on error.
+			if err := utils.LoadConfigFS(fsys); err == nil {
+				if functionConfig, ok := utils.Config.Functions[slug]; ok && !*functionConfig.VerifyJWT {
```

Comment:
Should we show an error message here if the config file cannot be loaded in here (parsing issues, etc) or is it handled elsewhere?

Otherwise users may wonder why their config isn't get picked up.

Reply:
Yup `LoadConfigFS` will do a syntax check on the config. But we need to write additional logic for handling semantic error like misspelt function name that doesn't exist in supabase/functions for eg.